prow/cluster/private/authentikos_deployment.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: authentikos
+  labels:
+    app: authentikos
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: authentikos
+  template:
+    metadata:
+      labels:
+        app: authentikos
+    spec:
+      serviceAccountName: authentikos
+      containers:
+      - name: authentikos
+        image: gcr.io/istio-testing/authentikos:0.0.1
+        imagePullPolicy: Always
+        args:
+        - --verbose
+        - --secret=authentikos-token
+        - --creds=/etc/creds/service-account.json
+        - --namespace=test-pods
+        - --scopes=https://www.googleapis.com/auth/devstorage.full_control
+        - "--format=Authorization: Bearer %v"
+        volumeMounts:
+        - name: creds
+          mountPath: /etc/creds
+          readOnly: true
+      volumes:
+      - name: creds
+        secret:
+          defaultMode: 0644
+          secretName: service-account
```

Comment:
nit: Consider creating a service account or token for this use case instead of reusing `service-account`.

Reply:
The usage of a `service-account` in the deployment *may* change if I can get **workload-identity** working (ref: https://github.com/istio/test-infra/pull/2106#discussion_r347760970). If this approach is not feasible, then I agree that an alternate SA (one scoped for the particular need) should be created.

Reply:
Looks like I was able to get  **workload-identity** working (in a scratch cluster) and verify that the `service-account` is no longer needed 🍾 ! I have updated the deployment to remove this volume and mount.